content/blog/2022-04-19-tracing/index.md

```diff
@@ -0,0 +1,142 @@
++++
+title = "Trace-Based Just-in-Time Type Specialization for Dynamic Languages"
+[extra]
+latex = true
+[[extra.authors]]
+name = "Yulun Yao"
++++
+
+## Introduction
+# TL.DR.
+This paper presents *Trace-Monkey*, a JavaScript language implementation based on tracing Just-In-Time (JIT) compilation. It focuses on identifying frequently executed loop traces at run-time and then compile those traces with dynamic type information. Specifically, the author proposes an algorithm to lazily form trace-trees with alternative paths within the loop, and explains how to generate efficient type-specialized code for the identified traces. The experimental result shows that *Trace-Monkey* is 10x faster compared to previous compilation strategies on some benchmarks.
```

Comment:
Please be consistent about whether it is Trace-Monkey or TraceMonkey. (Also, you only need to italicize it the first time.)